### ISSUE TITLE
Problem with routes

### ISSUE THREAD

**Issue by abdurahmanus** [Daniel15/RouteJs#51]
I have published my mvc6 site on IIS. Path to my site looks like http://mydomain/mysite.com. So, in my Startup.cs I made a trick:

```
public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory) {
	app.Map("/mysite", appBulder => RealConfigure(appBulder, env, loggerFactory));
}
public void RealConfigure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory) {
	....
}
```
All my urls are correct when I use @Url.Action("SomeAction", "SomeController") helper, but Router.action("SomeController", "SomeAction") gives me wrong url

**Comment by Daniel15** [comment]
Hmm, I'm using `HttpContext.Request.PathBase.Value` to obtain the base URL. It sounds like it doesn't contain `/mysite` like it should. I'll take a look!

**Comment by Daniel15** [comment]
Is the error that the URL is like `/mysiteFoo` rater than `/mysite/Foo` (ie. there's a slash missing)? That's the error I'm able to replicate. Is that what you're seeing, or are you seeing something else?

**Comment by abdurahmanus** [comment]
yes, url is like /mysiteFoo without slash

**Comment by Daniel15** [comment]
This should be fixed by e327483cb5777752c28ec38da6357a51eee2b78e. Can you please try the latest development version and see if it works properly for you? You can add https://ci.appveyor.com/nuget/routejs to your NuGet package sources to get a newer package.

**Comment by abdurahmanus** [comment]
Thank you for quick response. Now it works fine!

**Comment by Daniel15** [comment]
Great, thanks for confirming! I'll push an update to NuGet soon.

**Comment by Daniel15** [comment]
Fix pushed with 2.1 release.
